src/test/java/com/launchdarkly/testhelpers/httptest/RequestRecorderTest.java

```diff
@@ -21,16 +19,15 @@
 
 @SuppressWarnings("javadoc")
 public class RequestRecorderTest {
+  // Note that these tests are really testing two things: the RequestRecorder API, and the
+  // ability of the underlying server implementation to correctly get the request properties.
+  
```

Comment:
I reworked these tests a little just to make them more specifically test one feature at a time, which made it easier for me to validate the new server implementation.